### DISCUSSION TITLE
Why aren’t AMD Radeon 500 graphics cards supported?

### DISCUSSION THREAD

**Discussion by codingbutstillalive** [ROCm/ROCm#2185] [Q&A]
I wanted to use Apple's MPS, but it says it isn't supporting the 500 series of their AMD GPUs on Macbook Pros. However, these aren't so old. I bought my Macbook Pro in early 2019.

**Comment by MathiasMagnus** [comment]
AFAIK ROCm has nothing to do with anything going on in the Apple world. Apple provides all the SW support for anything that runs on their HW. ROCm doesn't support MacOS whatsoever, so the question of what generation of HW ROCm supports is irrelevant in the context of a MacBook Pro.

Having that said, on a personal level I too made the wrong bet with an ultra high-end laptop (GL702ZC) sporting a desktop RX 580, also gfx803 like many other 500 series GPUs. Support for it was dropped a little over 2-3 years after purchase. When the community got understandably furious, AMD took a stance that the _first_ gfx803 arch GPU was released 5+ years earlier, when support got dropped. Nobody cared that some of those GPUs were still on shelf. The gfx803 arch has powered 3-4 generations of GPUs (R9 Fury, 480, 580, 590), and supporting 590 (or the Macbook Pro) was exactly the same as supporting the Fury. In my personal opinion because it was in such a unique position, it should've been all the more reason to extend its lifespan, not scrap it under the rug.

None of this however applies to a Macbook Pro. Even though ROCm doesn't ship with gfx803 support out-of-the-box, 3rd party packaging of ROCm can enable it, as it's still kept around as an opt-in feature. Apple may just as well build their own binaries with 500-series support if they so wish (and I suspect they do), but AMD in their releases ended support for gfx803 some time ago. The R9 Fury was released in 2015, 8 years ago, but the 590 was released in 2018, just 5 years ago. [Heck, some brand new 580s are still on shelves](https://www.arukereso.hu/CategorySearch.php?st=radeon+rx+580). Somebody buying them buys it with instant out-of-support for ROCm. IMHO it should still be on life support, but the official support matrix says otherwise. (Even the following gfx900 arch powering the MI25 and consumer products like the Vega 64 has already been dropped.)

**Reply by codingbutstillalive** [reply]
Thank you really, really much. Yes, this stresses what I am feeling. Very disappointed by AMD as a company. And Apple not forcing them to behave otherwise.

**Reply by codingbutstillalive** [reply]
> AFAIK ROCm has nothing to do with anything going on in the Apple world. Apple provides all the SW support for anything that runs on their HW. ROCm doesn't support MacOS whatsoever, so the question of what generation of HW ROCm supports is irrelevant in the context of a MacBook Pro.
> 
> Having that said, on a personal level I too made the wrong bet with an ultra high-end laptop (GL702ZC) sporting a desktop RX 580, also gfx803 like many other 500 series GPUs. Support for it was dropped a little over 2-3 years after purchase. When the community got understandably furious, AMD took a stance that the _first_ gfx803 arch GPU was released 5+ years earlier, when support got dropped. Nobody cared that some of those GPUs were still on shelf. The gfx803 arch has powered 3-4 generations of GPUs (R9 Fury, 480, 580, 590), and supporting 590 (or the Macbook Pro) was exactly the same as supporting the Fury. In my personal opinion because it was in such a unique position, it should've been all the more reason to extend its lifespan, not scrap it under the rug.
> 
> None of this however applies to a Macbook Pro. Even though ROCm doesn't ship with gfx803 support out-of-the-box, 3rd party packaging of ROCm can enable it, as it's still kept around as an opt-in feature. Apple may just as well build their own binaries with 500-series support if they so wish (and I suspect they do), but AMD in their releases ended support for gfx803 some time ago. The R9 Fury was released in 2015, 8 years ago, but the 590 was released in 2018, just 5 years ago. [Heck, some brand new 580s are still on shelves](https://www.arukereso.hu/CategorySearch.php?st=radeon+rx+580). Somebody buying them buys it with instant out-of-support for ROCm. IMHO it should still be on life support, but the official support matrix says otherwise. (Even the following gfx900 arch powering the MI25 and consumer products like the Vega 64 has already been dropped.)

You may like to support this:

https://discussions.apple.com/thread/254892096
